src/contracts/Repository/Event/AfterEvent.php

```diff
@@ -12,6 +12,8 @@
 
 /**
  * Event emitted after action execution.
+ *
+ * @link https://github.com/symfony/symfony/blob/5.4/src/Symfony/Contracts/EventDispatcher/Event.php Symfony\Contracts\EventDispatcher\Event
```

Comment:
Maybe it would be nicer to have it inline in the description.

```suggestion
 * It extends {@link https://github.com/symfony/symfony/blob/5.4/src/Symfony/Contracts/EventDispatcher/Event.php Symfony\Contracts\EventDispatcher\Event}
```